pg_analytics/benchmarks/score.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
```

Comment:
could we write this in rust 🙏  don't want to introduce python tooling just for one file